.github/workflows/new-issues.yml

```
@@ -0,0 +1,19 @@
+name: 'Mark new issues with needs-triage label'
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+
+jobs:
+  label-new-issues
```

I believe this should be:

```suggestion
  label-new-issues:
```